### PR TITLE
Handle China National Day

### PR DESCRIPTION
**Pull request by kylekatarnls**
Partial fix for #58

### CODE REVIEW

**Review by liuyong888 — CHANGES_REQUESTED** [review]
@kylekatarnls

File: src/Cmixin/Holidays/cn-national.php [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-    '08-01-12:00'            => '= 08-01 12:00',
-    '10-01'                  => '10-01',
-    'spring-festival'        => '= chinese 01-01 - 1 day',
-    'chinese-01-0-01'        => '= chinese 01-01',
+    'new-year'                   => '01-01',
+    '03-08-12:00'                => '= 03-08 12:00',
+    '05-01'                      => '05-01',
+    '05-04-12:00'                => '= 05-04 12:00',
+    '06-01'                      => '06-01',
+    '08-01-12:00'                => '= 08-01 12:00',
```
[the remark this answers is not in the document]
Comment (kylekatarnls) [reply src/Cmixin/Holidays/cn-national.php:9]: Setting `'08-01-12:00' => null` make `isHoliday()` false, but `isWeekend()` will still be true, so `isBusinessDay()` should still be false if it's in the week-end.

But as it's not officially holiday, it's relevant to remove it.

File: src/Cmixin/Holidays/cn-national.php [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-    '03-08-12:00'            => '= 03-08 12:00',
-    '05-01'                  => '05-01',
-    '05-04-12:00'            => '= 05-04 12:00',
-    '06-01'                  => '06-01',
-    '08-01-12:00'            => '= 08-01 12:00',
-    '10-01'                  => '10-01',
-    'spring-festival'        => '= chinese 01-01 - 1 day',
-    'chinese-01-0-01'        => '= chinese 01-01',
+    'new-year'                   => '01-01',
+    '03-08-12:00'                => '= 03-08 12:00',
```
[the remark this answers is not in the document]
Comment (liuyong888) [reply src/Cmixin/Holidays/cn-national.php:5]: 03-08-12:00 Remove it!

**Comment by liuyong888** [comment]
When will it be merged into the master branch?
Thanks!
@kylekatarnls

**Comment by kylekatarnls** [comment]
You can now check the new list on `dev-master` version.

**Pull request merged**